backend/app/conversations/botVars.js

```diff
@@ -0,0 +1,25 @@
+
+const botVars = {
+  orgName: {
```

Comment:
My thought to put it in code was
- I thought it was easier to edit 
- we don't need to build a way to dynamically edit this initially

I was thinking in the future, when we make a settings page for admins, where they can dynamically edit these values, that would be a good time to move this into the database.

Basically, I think this could go in the db, but I don't think it would be difficult to change later